.eslintrc.json

```diff
@@ -2,7 +2,7 @@
   "root": true,
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 2022,
```

Comment:
Worth bumping to ES15 with 2024? Object.hasOwn and the array immutability changes are handy from ES14 as well as Promise.withResolvers from ES15.

ES16 should be next month so cool to rev with that.

Reply:
I will need to double check, as vscode/electron tends to lag a bit, compared to the browsers...

Reply:
Opened new ticket tcheck:  https://github.com/bytecodealliance/vscode-wit/issues/43